router/upload.go

```diff
@@ -109,7 +113,6 @@ func (f *UploadForm) ExtractInfo(r *http.Request) error {
 		return ErrInvalidTorrentCategory
 	}
 
-
```

Comment:
I just ran go fmt and it stripped the extra whitespace.  Not sure if trying to keep non-standard whitespace is sustainable at all.